auslib/blobs/schemas/systemaddons.yml

```diff
@@ -0,0 +1,72 @@
+---
+  title: SystemAddons Blob
+  type: object
+  required:
+    - name
+    - schema_version
+    - hashFunction
+    - addons
+  additionalProperties: false
+  properties:
+    name:
+      type: string
+      description: Human readable identifier
+      minLength: 1
+      maxLength: 100
+    schema_version:
+      type: number
+      description: Blob identifier number
+      enum:
+        - 5000
+    hashFunction:
+      type: string
+      description: Hash function of files pointed at (returned in responses generated by this Blob)
+    oneOf:
+        uninstall:
+          type: boolean
+          description: If this is set true, all system addons are disabled
+          default: false
```

Comment:
Does setting a default here make it impossible to have this value absent? If so, it may mean it's impossible to include any addons due to the oneOf requirement.